index.d.ts

```diff
@@ -1,3 +1,63 @@
+import { SnakeCase } from "type-fest";
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+type EmptyTuple = [];
+
+/**
+Return a default type if input type is nil.
+@template T - Input type.
+@template U - Default type.
+*/
+type WithDefault<T, U extends T> = T extends undefined | void | null ? U : T;
+
+/**
+Check if an element is included in a tuple.
+TODO: Remove this once https://github.com/sindresorhus/type-fest/pull/217 is merged.
```

Comment:
This was merged:

https://github.com/sindresorhus/type-fest/pull/217

Reply:
Thanks.
I tried to use [Includes](https://github.com/sindresorhus/type-fest/blob/main/source/includes.d.ts#L17) of `type-fest` for refactoring but failed.
This is because `Includes` is a bit different from `IsInclude` and the first one doesn't work.
So I revert  a commit (sorry for complicating commit log, should I recreate PR?)